assignment-tool/tool/src/webapp/vm/assignment/chef_assignments_instructor_new_edit_assignment.vm

```diff
@@ -29,14 +29,22 @@
 $(document).ready(function() {
  	setupAssignNew();
  	#if($turnitin_forceSingleAttachment)
+	 	var useReview = document.getElementById('$name_UseReviewService');
```

Comment:
turnitin_forceSingleAttachment is configurable via sakai.properties, this functionality was already existing. I just extended it and solved a bug.